elixir/apps/web/lib/web/live/settings/billing.ex

```diff
@@ -83,6 +85,21 @@ defmodule Web.Settings.Billing do
             </:value>
           </.vertical_table_row>
 
+          <.vertical_table_row :if={not is_nil(@account.limits.users_count)}>
+            <:label>
+              <p>Users</p>
+            </:label>
+            <:value>
+              <span class={[
+                not is_nil(@users_count) and
+                  @users_count > @account.limits.users_count && "text-red-500"
+              ]}>
+                <%= @users_count %> used
+              </span>
+              / <%= @account.limits.users_count %> allowed
+            </:value>
+          </.vertical_table_row>
+
```

Comment:
I think you need to change the section above, not add a new section here. Now we have a "Seats used" and "Users used" section

Reply:
@jamilbk Seats one will not be shown, just need to configure the plans properly after push to production and sync them with Firezone.